packages/ni.measurementlink.discovery.v1/README.md

```diff
@@ -0,0 +1,26 @@
+# Table of Contents
+
+- [Table of Contents](#table-of-contents)
+- [About](#about)
+  - [Operating System Support](#operating-system-support)
+  - [Python Version Support](#python-version-support)
+
+# About
+
+`ni.measurementlink.discovery.v1` is a Python package that provides data types for NI gRPC APIs in
+the [ni.measurementlink.discovery.v1 package](https://github.com/ni/ni-apis/tree/main/ni/measurementlink/discovery/v1).
+
+NI created and supports this package.
+
+## Operating System Support
+
+`ni.measurementlink.discovery.v1` supports Windows and Linux operating systems.
+
+## Python Version Support
+
+`ni.measurementlink.discovery.v1` supports CPython 3.9+.
```

Comment:
and pypy? or not? I think the other packages may be inconsistent here.

Cc: @jfriedri-ni @mjohanse-emr

Reply:
I'm fine with removing all references to pypy in this repo.

Reason: the protobuf and grpcio packages do not provide binary wheels for pypy. It may work, but you have to build both of these packages from source, and I don't want our PR workflows to do that.